src/components/controller/loader.ts

```diff
@@ -0,0 +1,49 @@
+import { IDataSources } from "../view/sources/sources";
+
+export type Callback<T> = (data: T) => void;
+class Loader {
+    baseLink: string;
+    readonly options: { [key: string]: string };
+
+    constructor(baseLink: string, options: { [key: string]: string }) {
```

Comment:
You can add access modifiers to the arguments for the constructor and in this case a separate field class declaration can be removed. For example,
constructor(private baseLink: string, private options: { [key: string]: string }) {...}

Reply:
Done, add access modifiers to the arguments for the constructor.